app/models/task.py

```diff
@@ -4,3 +4,29 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
+
+    def is_complete(self):
+        if self.completed_at:
+            return True
+        return False
```

Comment:
You can simplify this as:

```suggestion
    def is_complete(self):
        return bool(self.completed_at)
```